src/openms/include/OpenMS/KERNEL/FeatureMap.h

```diff
@@ -306,6 +307,12 @@ namespace OpenMS
 
     OPENMS_DLLAPI AnnotationStatistics getAnnotationStatistics() const;
 
+    OPENMS_DLLAPI std::set<IdentificationData::ObservationMatchRef> getUnassignedIDMatches() const;
```

Comment:
here as well... we might really need to group the api of the two different ID types clearly